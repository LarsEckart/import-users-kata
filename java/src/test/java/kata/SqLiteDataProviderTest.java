package kata;

import static org.assertj.core.api.Assertions.assertThat;

import java.util.List;
import org.approvaltests.Approvals;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

class SqLiteDataProviderTest {

  private DataProvider dataProvider;

  @BeforeEach
  void setUp() {
    dataProvider = new SqLiteDataProvider("test-users.db");
  }

  @Test
  void testSqLiteDataProviderToProvideUserInfoFromValidDbFile() {
    List<String[]> data = dataProvider.getData();

    Approvals.verifyAll("", data);
  }


}
