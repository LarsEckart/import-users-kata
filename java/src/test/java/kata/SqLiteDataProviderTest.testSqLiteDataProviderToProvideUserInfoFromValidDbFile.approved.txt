[0] = [202489617246, doesntMatter, Germany, 76694, Alice Smith, alice.smith@example.com, 1985-04-23T00:00Z[UTC]]
[1] = [202489618679, doesntMatter, USA, 90210, Bob Johnson, bob.johnson@example.com, 1990-07-15T00:00Z[UTC]]
[2] = [202489647246, doesntMatter, Spain, 28013, Carlos Diaz, carlos.diaz@example.com, 1978-12-05T00:00Z[UTC]]
[3] = [202489717311, doesntMatter, China, 100000, Diana Chen, diana.chen@example.com, 1995-03-10T00:00Z[UTC]]
